src/plugins/kibana/server/lib/__tests__/process_es_ingest_processors_response.js

```diff
@@ -0,0 +1,90 @@
+import processESIngestProcessorsResponse from '../process_es_ingest_processors_response';
+import expect from 'expect.js';
+import _ from 'lodash';
+
+describe('processESIngestSimulateResponse', function () {
+
+  it('should return a list of strings indicating the enabled processors', function () {
+    const response = {
+      nodes: {
+        node_foo: {
+          ingest: {
+            processors: [
+              { type: 'proc_foo' },
+              { type: 'proc_bar' }
+            ]
+          }
+        }
+      }
+    };
+
+    const expected = [ 'proc_foo', 'proc_bar' ];
+    const actual = processESIngestProcessorsResponse(response);
+
+    expect(actual).to.eql(expected);
```

Comment:
I still prefer to always use lodash to get deep and strict equality checking `expect(_.isEqual(actual, expected)).to.be.ok()`